pymoab-py3-18.04

```diff
@@ -0,0 +1,62 @@
+FROM  ubuntu:18.04
+
+# set timezone
+ENV HOME /root
+ENV TZ=America/Chicago
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+ENV HOME /root
+
+RUN apt-get -y --force-yes update
+RUN apt-get install -y --force-yes \
+    software-properties-common \
+    build-essential \
+    git \
+    cmake \
+    gfortran \
+    libblas-dev \
+    python3-pip \
+    liblapack-dev \
+    libhdf5-dev \
+    autoconf \
+    libtool
+
+# need to put libhdf5.so on LD_LIBRARY_PATH
+ENV LD_LIBRARY_PATH /usr/lib/x86_64-linux-gnu
+ENV LIBRARY_PATH /usr/lib/x86_64-linux-gnu
+
+# upgrade pip and install python dependencies
+ENV PATH $HOME/.local/bin:$PATH
+
+
+RUN python3 -m pip install --upgrade pip
+RUN pip install numpy scipy cython tables matplotlib setuptools future pytest
+
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 10; \
+    update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 10;
```

Comment:
maybe move this into the apt-get line after l22 ?
So you have all the setup together and you can change `python3` to `python` l32